controllers/util/provider.go

```diff
@@ -10,17 +11,21 @@ import (
 	"encoding/json"
 	"encoding/pem"
 	"fmt"
-	"os"
 	"time"
 
 	"github.com/google/uuid"
 	"github.com/red-hat-storage/ocs-operator/v4/services"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const onboardingValidationPrivateKeySecretName = "onboarding-private-key"
+
```

Comment:
I think this is also defined in onboarding-ticket-generator, could we make sure that we are not defining at only one place

Reply:
onboardingValidationPrivateKeySecretName is used at two places >
1) provider.go
2) onboarding-validation-keys-generator